docs/proposals/status.md

```diff
@@ -0,0 +1,126 @@
+## Background
+Application CRD has a list of GroupKinds and a LabelSelector to logically group live objects in the cluster. The Application CRD also provides a way to attach descriptive metadata to the logical grouping. 
+
+## What does this solve
+This PR proposes to enhance Application CRD controller to provide status tracking for the objects that match the selectors. This dramatically simplifies the status tracking requirements for a client that uses Application CRD. 
+
+## Proposal
+The proposal is to add a reconciler that creates watches for the GroupKinds that the application CRD refers to. The reconciler then inspects the matching objects and updates the aggregate status in the Application CRD.
+
+### New fields in Application CRD
+We propose to use Conditions to aggregate status of the matching objects.
+These conditions could be added to `.status.conditions`
+```
+"Ready"   //Aggregate readiness
+"Settled" //Observed generation is latest, the controller has acted on latest version of spec
```

Comment:
Can you describe in more details how `Settled` is computed

Reply:
please review https://github.com/kubernetes-sigs/application/pull/79
closing this